src/ol/events/SnapEvent.js

```diff
@@ -56,3 +57,9 @@ export class SnapEvent extends Event {
     this.segment = options.segment;
   }
 }
+
+export class UnsnapEvent extends Event {
+  constructor() {
+    super(SnapEventType.UNSNAP);
+  }
+}
```

Comment:
This event is not necessary - the `SnapEvent` can be constructed with `SnapEventType.UNSNAP` as first argument.